Neos.Neos/Resources/Private/Fusion/Prototypes/Augmenter.fusion

```diff
@@ -0,0 +1,13 @@
+# The Augmenter-component can be used as processor or as a standalone prototype
+
+prototype(Neos.Neos:Augmenter) < prototype(Neos.Fusion:Component) {
+  @class = 'PackageFactory\\AtomicFusion\\FusionObjects\\AugmenterImplementation'
```

Comment:
this should be `Neos\\Neos` and should have been caught in a test.. make sure not to test this with the Atomic.Fusion package installed..

Reply:
Fixed